docs/Usage/Storage Access Framework.md

```diff
@@ -514,6 +566,50 @@ final DocumentFile? createdFile = createFile(
 );
 ```
 
+### <samp>writeToFile</samp>
+
+<samp>Alias for `writeToFileAsBytes` and `writeToFileAsString`</samp>
+
+Convenient method to write to a file using `content` as `String` **or** `bytes` as `Uint8List`.
+
+You should provide either `content` or `bytes`, if both `bytes` will be used.
+
+`mode` represents the mode in which the file will be opened for writing. Use `FileMode.write` for truncating and `FileMode.append` for appending to the file.
+
+
+```dart
+final Uri documentUri = ...
+final String fileContent = 'My File Content';
+
+/// Write to a file using a [String] as file contents [content]
+final bool? success = writeToFile(
+  documentUri,
+  content: fileContent,
+  mode: FileMode.write,
+);
+
+/// Append to a file using a [String] as file contents [content]
+final bool? success = writeToFile(
+  documentUri,
+  content: fileContent,
+  mode: FileMode.append,
+);
+
+/// Write to a file using a [Uint8List] as file contents [bytes]
+final DocumentFile? createdFile = writeToFile(
+  documentUri,
+  content: Uint8List.fromList(fileContent.codeUnits),
+  mode: FileMode.write,
+);
+
+/// Append to a file using a [Uint8List] as file contents [bytes]
+final DocumentFile? createdFile = writeToFile(
```

Comment:
```suggestion
final bool? success = writeToFile(
```

Guess this function returns a `boolean` representing the whether or not the operation was successful, if not let me know but I think it was just a typo.